### PR TITLE
Add What's New in v1.10.0

### PR DESCRIPTION
**Pull request by dennisreimann**
grafik

### CODE REVIEW

File: BTCPayServer/Views/UIStores/Dashboard.cshtml [hunk 1]
```diff
@@ -34,17 +34,17 @@
                 </button>
             </div>
             <div class="modal-body">
+                <h5 class="alert-heading">Updated in <a href="https://blog.btcpayserver.org/btcpay-server-1-10-0/" target="_blank" rel="noreferrer noopener">v1.10.0</a></h5>
+                <p class="mb-2">We've added some more options for refunds and an editor for creating custom forms. A propos custom: You can now also define your own roles with customized access to store features.</p>
```
Comment (Bas02) [inline BTCPayServer/Views/UIStores/Dashboard.cshtml:38]:
```suggestion
                <p class="mb-2">We've added some more options for refunds and an editor for creating custom forms. You can now also define your own roles with customized access to store features.</p>
```

**Review by Bas02 — CHANGES_REQUESTED** [review]
Couldn't figure out what you tried to say with what i took out.

**Comment by dstrukt** [comment]
`A propos custom` is less common in the states, usually see `with regard to`, or `speaking of...`

Might consider dropping, or editing, but just my 2sats, otherwise, looks great!

**Comment by dennisreimann** [comment]
Yup, @Bas02 already took that out, good to have two second pairs of eyes on it 😁

**Review by pavlenex — APPROVED** [review]

**Pull request merged**